src/modules/header/styles.css

```diff
@@ -266,3 +268,70 @@
   font-size: .875rem;
   color: hsl(var(--app-bar-color, var(--component-color) / var(--component-medium-emphasis)));
 }
+
+
+/* search item styles */
+.search-bar-item {
+  --app-bar-padding-y: .25rem;
+
+  flex: 1;
+}
+
+.select {
+  --select-background: var(--gray-200);
+  --select-focused-background: var(--white);
+  --selected-option-color: var(--gray-200);
+  --form-control-padding-y: .5rem;
+  --form-control-color: var(--gray-700);
+
+  max-width:
+    calc(
+      (100vw - 2 * var(--page-padding-x)) * var(--main-content-percentage-width)
+    );
+
+  padding: calc(.25rem - 1px) .25rem .25rem;
+}
+
+.select ul > * {
+  padding: .25rem;
+}
+
+.select:focus-within {
+  --form-control-color: var(--primary);
+
+  border-bottom-right-radius: 0;
+  border-bottom-left-radius: 0;
+  box-shadow: 0 2px 16px rgba(0, 0, 0, .25), 0 2px 4px rgba(0, 0, 0, .05);
+}
+
+.select:focus-within div:nth-child(2) {
+  border-bottom: 1px solid var(--primary);
+}
+
+.select div:nth-child(2) > * {
+  border: none;
+}
+
+.select label {
+  font-weight: 400;
+  color: var(--black);
+}
+
+@media (--header-breakpoint-down) {
+  .select {
+    max-width: 100%;
+  }
+
+  .select input {
+    width: 1rem;
+  }
+
+  .select:focus-within {
+    position: absolute;
+    top: var(--app-bar-padding-y);
+    right: .5rem;
+    bottom: var(--app-bar-padding-y);
+    left: .5rem;
+    max-width: calc(100vw - var(--page-padding-x));
+  }
+}
```

Comment:
Is it possible to rewrite this using a mobile-first approach? Also, joining media queries with the selectors themselves, i.e. incapsulating queries into selectors so they are not repeated?